### PR TITLE
refactored binding.gyp

### PR DESCRIPTION
**Pull request by refack**
Will compile on windows with `node-gyp:3.6.0`

### CODE REVIEW

**Comment by addaleax** [comment]
Awesome, thank you a lot! Can you edit the CI files to turn `npm install node-gyp@3.5.0` into `npm install node-gyp@^3.6.0`, too? If not I’m happy to just do that myself.

**Comment by refack** [comment]
Done. Now I'm curious "will it blend" on 3.5.0...

File: appveyor.yml [hunk 1]
```diff
@@ -19,14 +19,11 @@ environment:
 
 
 install:
-  # From https://github.com/nodejs/nan/blob/master/appveyor.yml
-  - ps: Install-Product node $env:nodejs_version
-  - ps: Update-NodeJsInstallation (Get-NodeJsLatestBuild $env:nodejs_version) $env:platform
```
Comment (refack) [inline appveyor.yml]: AppVeyor now does it on it's own according to `nodejs_version`

**Comment by refack** [comment]
Guess not https://ci.appveyor.com/project/addaleax/lzma-native/build/239/job/sv3uuu30fbocaqyt

File: appveyor.yml (hunk 1 above)
Comment (refack) [inline appveyor.yml]: https://www.appveyor.com/docs/lang/nodejs-iojs/#testing-under-multiple-versions-of-nodejs-or-iojs

File: binding.gyp [hunk 2]
```diff
@@ -55,22 +55,33 @@
...
+            }, {
+              'variables': {
+                "arch_lib_path" : 'bin_i686',
+                "arch_lib_code" : 'ix86'
+              }
+            } ]
+          ],
           "actions" : [
             {
+              "msvs_quote_cmd": 0,
```
Comment (refack) [inline binding.gyp]: This kills the regression in `node-gyp`

**Comment by refack** [comment]
I assume the red travis is an encrypted value permission issue:
```
$ openssl aes-256-cbc -K $encrypted_5d419efedfca_key -iv $encrypted_5d419efedfca_iv -in .travis.id_rsa.enc -out .travis.id_rsa -d
iv undefined
The command "openssl aes-256-cbc -K $encrypted_5d419efedfca_key -iv $encrypted_5d419efedfca_iv -in .travis.id_rsa.enc -out .travis.id_rsa -d" failed and exited with 1 during .
```

**Comment by refack** [comment]
@addaleax landing?

**Comment by addaleax** [comment]
Yeah, merging. Thanks for the hard work you’ve been putting into node-gyp & everything! :heart:

**Pull request merged**